Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -32,39 +32,36 @@ public NoOpRunner(IConnectionManager connectionManager)
 
             /* SHAPE FACTORY DESIGN PATTERN */
             ShapeFactory shapeFactory = new ShapeFactory();
-            BaseShape shape1 = shapeFactory.GetShape(Shape.Square, 5, 5);
-            BaseShape shape2 = shapeFactory.GetShape(Shape.Circle, 10, 5);
-            BaseShape shape3 = shapeFactory.GetShape(Shape.Rectangle, 15, 5);
+            BaseShape shape1 = shapeFactory.GetShape(Shape.HealthCrystal, 1, 15);
+            BaseShape shape2 = shapeFactory.GetShape(Shape.DamageCrystal, 5, 15);
+            BaseShape shape3 = shapeFactory.GetShape(Shape.Saw, 8, 15);
             GamePlatforms.AddShape(shape1);
             GamePlatforms.AddShape(shape2);
             GamePlatforms.AddShape(shape3);
 
             /* ABSTRACT SHAPE FACTORY DESIGN PATTERN */
-            AbstractFactory abstractShapeFactory = FactoryProducer.GetFactory(true);
-            BaseShape aShape1 = abstractShapeFactory.GetShape(Shape.Stairs, 20, 5);
-            BaseShape aShape2 = abstractShapeFactory.GetShape(Shape.Stone, 22, 5);
-            BaseShape aShape3 = abstractShapeFactory.GetShape(Shape.Fence, 24, 5);
+            AbstractFactory abstractShapeFactory = FactoryProducer.GetFactory(passable: true);
+            BaseShape aShape1 = abstractShapeFactory.CreateStaticShape(Shape.Platform, 0, 0, 0, 10);
+            BaseShape aShape2 = abstractShapeFactory.CreateEntityShape(Shape.HealthCrystal, 10, 15);
+            BaseShape aShape3 = abstractShapeFactory.CreateEntityShape(Shape.DamageCrystal, 15, 15);
             GamePlatforms.AddShape(aShape1);
             GamePlatforms.AddShape(aShape2);
             GamePlatforms.AddShape(aShape3);
 
-            /*Platform firstPlatform = new Platform(0, 0, 0, 10);
-            GameWindow.AddShape(firstPlatform); // Main platform
+            /*AbstractFactory inpassableFactory = FactoryProducer.GetFactory(false);
```

Comment:
Same as before, don't pass bool literals like that, use named arguments (`passable: true`). Same for one other place below.

Reply:
Fixed